app/src/main/java/com/infomaniak/mail/data/cache/RealmMigrations.kt

```diff
@@ -87,6 +88,24 @@ private fun MigrationContext.keepDefaultValuesAfterSixthMigration() {
     }
 }
 
+// Migrate from version #9
+private fun MigrationContext.renameKSuiteRelatedBooleans() {
+
+    if (oldRealm.schemaVersion() <= 9L) {
+        enumerate(className = "Mailbox") { oldObject: DynamicRealmObject, newObject: DynamicMutableRealmObject? ->
+            newObject?.apply {
+
+                // Rename property without losing its previous value
+                set(propertyName = "isKSuitePerso", value = oldObject.getValue<Boolean>(fieldName = "isFree"))
+
+                // Rename property without losing its previous value
+                set(propertyName = "isKSuitePersoFree", value = oldObject.getValue<Boolean>(fieldName = "isLimited"))
```

Comment:
Does this have any chance of crashing if a version without the "is_free" and "is_limited" tries to migrate to this version ? 
I think it could